build.sh

```diff
@@ -0,0 +1,11 @@
+# Install root packages
+npm install
+
+# Install backend packages
+cd packages/cfa-template/template/backend
```

Comment:
With `workspaces` (good addition to the template) in place, you shouldn't need to cd into frontend and backend to install... `npm` ought to have installed dynamically down the chain. Do you see something different?